lua/toggleterm/commandline.lua

```diff
@@ -93,8 +93,13 @@ local term_exec_options = {
 
     for _, path in ipairs(paths) do
       -- solve space in file names
-      local glob_str = path.gsub(" ", "\\ ")
+      if path != '' 
```

Comment:
This isn't valid `lua` you probably meant to write `~=`

Reply:
Sorry about the inconvenience and I am happy to be a part of your project